_config.yml

```diff
@@ -1,37 +1,15 @@
-# Welcome to Jekyll!
-#
-# This config file is meant for settings that affect your whole blog, values
-# which you are expected to set up once and rarely edit after that. If you find
-# yourself editing this file very often, consider using Jekyll's data files
-# feature for the data you need to update frequently.
-#
-# For technical reasons, this file is *NOT* reloaded automatically when you use
-# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
-
 # Site settings
 title: OffenerHaushalt
-
 email: info@okfn.de
-
-description: ''
-
-# the subpath of your site, e.g. /blog
+description: 'Haushaltsdaten von Bund, Ländern und Kommunen einfach verständlich präsentieren.'
 baseurl: ''
-
-# the base hostname & protocol for your site, e.g. http://example.com
 url: 'https://offenerhaushalt.de'
 
-twitter_username: okfde
-github_username:  okfde
-
 # Haushalte
 collections:
   budget:
     output: true
     permalink: /ohaushalt/:name/
-  haushalte:
```

Comment:
I am 99% shure its correct to remove this. It was commented out before. Then in https://github.com/okfde/offenerhaushalt.de/commit/06108006ac0d25aa65ebed738ea3c432ff87354c#diff-aeb42283af8ef8e9da40ededd3ae2ab2 both "haushalte:" where activated, which was an error IMO. So now I removed the one that was commented out before, so there is no missunderstanding anymore for the future.